eclipse-templatevariables-feature/feature.xml

```diff
@@ -14,68 +14,68 @@
    </copyright>
 
    <license>
-                                 Apache License
+      Apache License
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
 
    TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
 
    1. Definitions.
 
-      "License" shall mean the terms and conditions for use, reproduction,
+      &quot;License&quot; shall mean the terms and conditions for use, reproduction,
```

Comment:
was that automatically changed by your XML formatter?

Reply:
I do not care much, but just wanted to understand if it is wrong to use regular quotes here (according to my XML understanding this is fine and `&quot;` is only required in double quoted attribute values.)

Reply:
It was automatically changed by eclipse. Following https://stackoverflow.com/questions/150423/whats-the-accepted-way-of-storing-quoted-data-in-xml BOTH ways are correct. I would suggest to used ``&quot``. Let me know...

Reply:
Sure. Fine this way. Just wanted to understand this...